bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -3517,18 +3518,12 @@ public void createPartControl(final Composite parent) {
 
    private Section createSection(final Composite parent,
                                  final FormToolkit tk,
-                                 final String title,
-                                 final boolean isGrabVertical,
-                                 final boolean isExpandable) {
```

Comment:
Because I wanted to fix the bug below where even though the section `Tour Event` is collapsed, there is still a lot of empty space displayed

Top is with my fix
Bottom is MT 24.1

![image](https://github.com/mytourbook/mytourbook/assets/25235665/b886cf24-d680-4224-8478-670bc3c04417)

Reply:
> Because I wanted to fix the bug below where even though the section `Tour Event` is collapsed, there is still a lot of empty space displayed

For me this was not a bug. 

I adjusted the code to grab vertial space only when the tour section is expanded https://github.com/mytourbook/mytourbook/commit/4589fe62165311db96cf71b486f0d27b42b0fbd2